build/Helix/runtests.cmd

```diff
@@ -12,7 +12,7 @@ cd scripts
 powershell -ExecutionPolicy Bypass .\InstallTestAppDependencies.ps1
 cd ..
 
-set testBinaryCandidates=MUXControls.Test.dll MUXControlsTestApp.appx IXMPTestApp.appx MUXControls.ReleaseTest.dll NugetPackageTestApp.appx NugetPackageTestAppCX.appx
```

Comment:
if `te` doesn't need to specify appx, should we remove MUXControlsTestApp.appx IXMPTestApp.appx too?

Reply:
No. Those appxes contain tests (the ApiTests, etc.). NugetPackageTestApp.appx and NugetPackageTestAppCX.appx don't contain any tests. MUXControls.ReleaseTest.dll will install and launch these apps and interact with them. But the apps themselves do not contain any tests that TAEF can execute.